proposals/lip-0055.md

```diff
@@ -97,7 +97,7 @@ For the rest of this proposal we define the following constants.
 | `MAX_MODULE_NAME_LENGTH` | integer | 32 | The maximum length of a string specifying the name of a module. |
 | `OWN_CHAIN_ID` | `bytes` | set in configuration | The chain ID of the chain under consideration as defined in [LIP 0037][lip-0037#own-chain-id]. |
 
-Furthermore, in the following we indicate with `block` be the block under consideration and with `previousBlock` the previous block of the chain. Calling a function `fct` from another module `module` is represented by `module.fct`.
+Furthermore, in the following we indicate with `block` be the block under consideration and with `previousBlock` the previous block of the chain. Calling a function `fct` from another module `module` is represented by `module.fct`. The functions `decode` and `validateObjectSchema` are defined in [LIP 0027][https://github.com/LiskHQ/lips/blob/main/proposals/lip-0027.md#notation-for-encoding-and-decoding].
```

Comment:
I would prefer to define a new link like the others, by adding it to the bottom of the LIP.